samples/src/main/java/com/example/bigquery/CopyMultipleTables.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquery;
+
+// [START bigquery_copy_table_multiple_source]
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.CopyJobConfiguration;
+import com.google.cloud.bigquery.Job;
+import com.google.cloud.bigquery.JobInfo;
+import com.google.cloud.bigquery.TableId;
+import java.util.Arrays;
+
+public class CopyMultipleTables {
+
+  public static void runCopyMultipleTables() {
+    // TODO(developer): Replace these variables before running the sample.
+    String destinationDatasetName = "MY_DATASET_NAME";
+    String destinationTableId = "MY_TABLE_NAME";
+    copyMultipleTables(destinationDatasetName, destinationTableId);
+  }
+
+  public static void copyMultipleTables(String destinationDatasetName, String destinationTableId) {
+    try {
+      // Initialize client that will be used to send requests. This client only needs to be created
+      // once, and can be reused for multiple requests.
+      BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
+
+      TableId destinationTable = TableId.of(destinationDatasetName, destinationTableId);
+
+      // For more information on CopyJobConfiguration see:
+      // https://googleapis.dev/java/google-cloud-clients/latest/com/google/cloud/bigquery/JobConfiguration.html
+      CopyJobConfiguration configuration =
+          CopyJobConfiguration.newBuilder(
+                  destinationTable,
```

Comment:
note the eight space indent here. This is inconsistent with google style that specifies four spaces for a continuation line. It's the same thing I saw this morning in a different PR and filed #72 about.

I wonder if we need to update our version of something related to formatting?

Reply:
Yes - I raised a ticket with the team responsible for the tool (coveo). We do have the latest version (2.9). I will keep you posted.